csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -124,5 +97,41 @@ public sealed class OutgoingRequestFrame : OutputStream
 
             ContextHelper.Write(this, Context);
         }
+
+        /// <summary>Creates a new outgoing request frame with the given payload.</summary>
+        /// <param name="proxy">A proxy to the target Ice object. This method uses the communicator, identity, facet
+        /// and context of this proxy to create the request frame.</param>
+        /// <param name="operation">The operation to invoke on the target Ice object.</param>
+        /// <param name="idempotent">True when operation is idempotent, otherwise false.</param>
+        /// <param name="context">An optional explicit context. When non null, it overrides both the context of the
+        /// proxy and the communicator's current context (if any).</param>
+        /// <param name="payload">The payload of this request frame, which represents the marshaled in-parameters.
+        /// </param>
+        public OutgoingRequestFrame(IObjectPrx proxy, string operation, bool idempotent, Context? context,
+                                    ArraySegment<byte> payload)
+            : this(proxy, operation, idempotent, context)
+        {
+            if (payload.Count == 0)
+            {
+                WriteEmptyEncapsulation(proxy.EncodingVersion);
+            }
+            else
+            {
+                WritePayload(payload);
+            }
+        }
+
+        /// <summary>Starts writing the parameters for this request.</summary>
+        /// <param name="format">The format for the payload, SlicedFormat or CompactFormat.</param>
+        public void StartParameters(FormatType? format = null)
+        {
+            StartEncapsulation(_payloadEncoding, format);
+        }
```

Comment:
Use expression body instead
```suggestion
        public void StartParameters(FormatType? format = null) => StartEncapsulation(_payloadEncoding, format);
```

Reply:
Fixed